server/test/recipesController.spec.js

```diff
@@ -1,166 +1,257 @@
+          expect(res).to.have.status(404);
+          done();
+        });
+    });
 
-  it('should return 204 if recipe is updated successfully', (done) => {
-    chai.request(app)
-      .put(`/api/v1/recipes/${recipe}`)
-      .set('Content-Type', 'application/json')
-      .set('x-access-token', userToken)
-      .send({
-        recipeName: faker.name.findName(),
-        ingredient: 'new lorem, ipsum, itsum, gallum',
-        recipeDirection: 'This is the updated sample recipe description',
-        recipeImage: 'http://www/allnigeriafoods.com/sample-recipe-url'
-      })
-      .end((err, res) => {
-        expect(res).to.have.status(204);
-        done();
-      });
-  });
+  it(`should return 'internal server error' 
+  if wrong recipe ingredient name is supplied`, (done) => {
+      chai.request(app)
+        .put(`/api/v1/recipes/${recipeId}`)
+        .set('Content-Type', 'application/json')
+        .set('x-access-token', userToken)
+        .send({
+          name: 'sushi',
+          wrongIngredientNameVariable: 'new lorem, ipsum, itsum, gallum',
+          direction: `This is a sample recipe description. 
+        If you parboiled the rice as described at
+        parboiling rice for cooking jollof rice,
+         the rice should be done by the time the water is dry. 
+         Taste to confirm. If not, you will need to add more water and 
+         educe the heat to prevent burning. Keep cooking till done.`,
+          image: 'http://www/allnigeriafoods.com/sample-recipe-url'
+        })
+        .end((err, res) => {
+          expect(err.message).to.equal('Internal Server Error');
+          expect(res).to.have.status(500);
+          done();
+        });
+    });
 
-  it('should return 400 if recipe update is not successfully', (done) => {
+  it('should return 200 if recipe is updated successfully', (done) => {
     chai.request(app)
-      .put(`/api/v1/recipes/${recipe}`)
+      .put(`/api/v1/recipes/${recipeId}`)
       .set('Content-Type', 'application/json')
       .set('x-access-token', userToken)
       .send({
-        recipeName: faker.name.findName(),
-        wrongIngredientName: 'new lorem, ipsum, itsum, gallum',
-        recipeDirection: 'This is the updated sample recipe description',
-        recipeImage: 'http://www/allnigeriafoods.com/sample-recipe-url'
+        name: 'sushi',
+        ingredients: 'updated lorem, ipsum, itsum, gallum',
+        direction: `This is a sample recipe description. 
+        If you parboiled the rice as described at
+        parboiling rice for cooking jollof rice,
+         the rice should be done by the time the water is dry. 
+         Taste to confirm. If not, you will need to add more water and 
+         educe the heat to prevent burning. Keep cooking till done.`,
+        image: 'http://www/allnigeriafoods.com/sample-recipe-url'
       })
       .end((err, res) => {
-        expect(res).to.have.status(400);
+        expect(res.body.message).to.equal('Recipe updated');
```

Comment:
@fuchodeveloper Test that the recipe is actually updated by confirming that it's name/ingredients/direction etc is as expected.